samples/commands/execute_command/README.md

```diff
@@ -0,0 +1,529 @@
+and receive as necessary to perform all of the data plane operations. Below is a sample policy that can be used on your
+IoT Core Thing that will allow this sample to run as intended.
+
+<details>
+<summary>Sample Policy</summary>
+<pre>
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": "iot:Publish",
+      "Resource": [
+        "arn:aws:iot:&lt;region&gt;:&lt;account&gt;:topic/$aws/commands/&lt;device_type&gt;/&lt;device_id&gt;/executions/*/response/json"
+      ]
+    },
+    {
+      "Effect": "Allow",
+      "Action": "iot:Receive",
+      "Resource": [
+        "arn:aws:iot:&lt;region&gt;:&lt;account&gt;:topic/$aws/commands/&lt;device_type&gt;/&lt;device_id&gt;/executions/*/request/*",
+        "arn:aws:iot:&lt;region&gt;:&lt;account&gt;:topic/$aws/commands/&lt;device_type&gt;/&lt;device_id&gt;/executions/*/response/accepted/json",
+        "arn:aws:iot:&lt;region&gt;:&lt;account&gt;:topic/$aws/commands/&lt;device_type&gt;/&lt;device_id&gt;/executions/*/response/rejected/json"
+      ]
+    },
+    {
+      "Effect": "Allow",
+      "Action": "iot:Subscribe",
+      "Resource": [
+        "arn:aws:iot:&lt;region&gt;:&lt;account&gt;:topicfilter/$aws/commands/&lt;device_type&gt;/&lt;device_id&gt;/executions/*/request/*",
+        "arn:aws:iot:&lt;region&gt;:&lt;account&gt;:topicfilter/$aws/commands/&lt;device_type&gt;/&lt;device_id&gt;/executions/*/response/accepted/json",
+        "arn:aws:iot:&lt;region&gt;:&lt;account&gt;:topicfilter/$aws/commands/&lt;device_type&gt;/&lt;device_id&gt;/executions/*/response/rejected/json"
+      ]
+    },
+    {
+      "Effect": "Allow",
+      "Action": "iot:Connect",
+      "Resource": "arn:aws:iot:&lt;region&gt;:&lt;account&gt;:client/test-*"
+    }
+  ]
+}
+</pre>
+
+Replace with the following with the data from your AWS account:
+* `<region>`: The AWS IoT Core region where you created your AWS IoT Core thing you wish to use with this sample. For example `us-east-1`.
+* `<account>`: Your AWS IoT Core account ID. This is the set of numbers in the top right next to your AWS account name when using the AWS IoT Core website.
+* `<device_type>`: Can be either `things` or `clients`.
+* `<device_id>`: Depending on `<device_type>` value, this is either IoT Thing name or MQTT client ID.
+
+Note that in a real application, you may want to avoid the use of wildcards in your ClientID or use them selectively.
+Please follow best practices when working with AWS on production applications using the SDK. Also, for the purposes of
+this sample, please make sure your policy allows a client ID of `test-*` to connect or use `--client_id <client ID here>`
+to send the client ID your policy supports.
+
+</details>
+
+Additionally, the AWS CLI triggered control plane operations in the walkthrough require that AWS credentials with
```

Comment:
"Additionally, the AWS CLI triggered control plane operations in the walkthrough require that AWS credentials with appropriate permissions be sourceable. At a minimum, the following permissions must be granted:"
->
"The AWS CLI triggered control plane operations in the walkthrough require AWS credentials with appropriate permissions be sourceable. The following permissions must be granted:"